### ISSUE TITLE
Midpoint Review - IA Feedback - Benefits Team 1, Supplemental Claims Form 20-0995

### ISSUE THREAD

**Issue by jimuhh** [department-of-veterans-affairs/va.gov-team#46611]
## VFS acceptance criteria
- [ ] Let Platform know when the **Must** feedback has been incorporated 
- [ ] Leave any comments for feedback you decide _not_ to take
- [ ] VFS team closes the ticket

## Thoughts/questions

- If the degree of content oversight difficulty on a static page on a familiar VA topic is 3 or 4, this form is a full-on Nigel Tufnel-level 11. Leading users through an unfamiliar process with strange terminology and confusing-yet-crucial distinctions is one of the hardest content jobs possible. This isn't news to you. Your product documentation shows full awareness of the challenges. 
- With that in mind, I reached out to Danielle Thierry at VA content to let her know that I'm recommending an extra level of content diligence in view of the difficult challenge. 
- I'm going to let her take it from here. Any specific content advice I offer below is strictly "consider." 

## Feedback
- Practice areas will document their feedback following the [Must, Should, and Consider Framework](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/must-should-and-consider-framework-for-feedback).

**Must:**

- Complete sitewide [content and IA intake](https://github.com/department-of-veterans-affairs/va.gov-team/issues/43919#:~:text=sitewide%20content%20and%20IA%20intake%20request%20process) for end-to-end process. 
- Work closely with Sitewide Content and IA to fine-tune instructions for scannability and plain language 

**Should:**

**Consider:**

- Applying strategic bulleting and bolding to crucial info as per [Nielsen Norman guidelines](https://www.nngroup.com/articles/how-users-read-on-the-web/)
- Testing revised instructions for readability and comprehension on key info.
  - e.g.: "What do you always need to supply when filing a supplemental claim?"
- User testing your content with the UK.gov [simple test for evaluating content](https://userresearch.blog.gov.uk/2014/09/02/a-simple-technique-for-evaluating-content/)
## Platform directions
- Remove the governance-team label
- Update "Issue Title"
- Link to collab cycle Request epic
- Add your feedback
- Add assignees based on collab cycle touchpoint
  - **Design Intent**: VFS designer, VFS PM (optional), yourself (optional)
  - **Midpoint Review**: VFS PM, yourself (optional)
  - **Staging Review QA Only**: VFS PM, yourself (optional)

**Comment by thehastingsp** [comment]
@jimuhh thank you for your feedback. I've created a source of truth and a sitewide content/IA ticket to review for plain language end to end. I did work with @bethpotts  and team on several of the pieces, but I do think it is prudent to view the entire conversation. 

MUST: 

- [x] `Complete sitewide content and IA intake process for end-to-end process.`

Ticket is [here ](https://github.com/department-of-veterans-affairs/va.gov-team/issues/46818)

- [ ] Work closely with Sitewide Content and IA to fine-tune instructions for scannability and plain language. 

Doing
